### PR TITLE
stun nettle introduction

### PR DESCRIPTION
**Pull request by danhibiki**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs )

:cl: 
add: Added stun nettles. Credit to alehkin for all the icons
/:cl:

[why]: Because I want to make non-lethal variant of stun nettles that people can use to incapacitate but not kill someone and do it slowly. Using tirizene and tiring solution will make it slower than using a cattleprod or a stun baton but its more inline with botany

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
>more stuns
into the trash it goes

**Comment by ghost** [comment]
hey its more like paralysis nettle?  paranettle,  tiring nettle kinda shit.  not instant,  more like date rape

**Comment by danhibiki** [comment]
Its by tirizene and tiring solution, I admit para nettle might be a bit more accurate but I figured stun nettle looks a bit better in my opinion. not against changing the name if it makes or breaks it

**Comment by Lxtoofonea** [comment]
At the moment I could swing either way

**Comment by bardofrage** [comment]
At this point, combat is already dominated by stuns to the point of everyone having them, might as well give botany some utility at least. +1

File: hippiestation/code/modules/hydroponics/grown/nettle.dm [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/obj/item/seeds/nettle
+	mutatelist = list(/obj/item/seeds/nettle/death, /obj/item/seeds/nettle/stun)
```
Comment (k-k-karma) [inline hippiestation/code/modules/hydroponics/grown/nettle.dm]: new line here please

File: hippiestation/code/modules/hydroponics/grown/nettle.dm [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	icon_state = "seed-stunnettle"
+	species = "stunnettle"
+	plantname = "Stun Nettles"
+	product = /obj/item/reagent_containers/food/snacks/grown/nettle/stun
+	maturation = 8
+	yield = 2
+	genes = list(/datum/plant_gene/trait/repeated_harvest, /datum/plant_gene/trait/plant_type/weed_hardy, /datum/plant_gene/trait/stinging)
+	mutatelist = list()
+	reagents_add = list("tirizene" = 0.2, "tiresolution" = 0.2, "pax" = 0.05, "kelotane" = 0.05)
+	rarity = 20
```
Comment (k-k-karma) [inline hippiestation/code/modules/hydroponics/grown/nettle.dm]: another one here

File: hippiestation/code/modules/hydroponics/grown/nettle.dm (hunk 1 above)
Comment (danhibiki) [reply hippiestation/code/modules/hydroponics/grown/nettle.dm:2]: For the mutate list? This is in line with the standard that all other seeds use to declare mutation types exceeding one for all I saw

File: hippiestation/code/modules/hydroponics/grown/nettle.dm (hunk 2 above)
Comment (danhibiki) [reply hippiestation/code/modules/hydroponics/grown/nettle.dm:17]: I’m lost on this one. What do you mean

File: hippiestation/code/modules/hydroponics/grown/nettle.dm (hunk 1 above)
Comment (k-k-karma) [reply hippiestation/code/modules/hydroponics/grown/nettle.dm]: no after the item definition

Comment (k-k-karma) [reply hippiestation/code/modules/hydroponics/grown/nettle.dm]:
```suggestion
	mutatelist = list(/obj/item/seeds/nettle/death, /obj/item/seeds/nettle/stun)
	
```

Comment (danhibiki) [reply hippiestation/code/modules/hydroponics/grown/nettle.dm:2]: Got it

File: hippiestation/code/modules/hydroponics/grown/nettle.dm (hunk 2 above)
Comment (k-k-karma) [reply hippiestation/code/modules/hydroponics/grown/nettle.dm]:
```suggestion
	rarity = 20
	
```

**Comment by RafRoq** [comment]
/datum/plant_gene/trait/stinging this trait does no inject chemicals on attack only on throw it must be changed

**Comment by Lxtoofonea** [comment]
+1, every department right now can get stuns, is so easy for a botanist to just run into robotics and grab a flash, it will take some time for someone to make this.

**Comment by ghost** [comment]
merge?

**Pull request merged**